Doc/whatsnew/3.13.rst

```diff
@@ -79,6 +79,13 @@ Other Language Changes
 * Allow the *count* argument of :meth:`str.replace` to be a keyword.
   (Contributed by Hugo van Kemenade in :gh:`106487`.)
 
+* Compiler now strip indents from docstrings.
+  This will reduce size of ``.pyc`` cache files.
```

Comment:
```suggestion
  This will reduce the size of ``.pyc`` (cached bytecode) files.
```

Could probably make `pyc` a link to glossary or import docs too!